modulefiles/build_hera_gnu.lua

```diff
@@ -0,0 +1,27 @@
+help([[
+This module loads libraries for building the UFS SRW App on
+the NOAA RDHPC machine Hera using GNU 9.2.0
+]])
+
+whatis([===[Loads libraries needed for building the UFS SRW App on Hera using GNU 9.2.0 ]===])
+
+prepend_path("MODULEPATH","/contrib/sutils/modulefiles")
+load("sutils")
+
+load(pathJoin("cmake", os.getenv("cmake_ver") or "3.20.1"))
+
+prepend_path("MODULEPATH", "/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/modulefiles/stack")
+
+load(pathJoin("hpc", os.getenv("hpc_ver") or "1.2.0"))
+load(pathJoin("hpc-gnu", os.getenv("hpc-gnu_ver") or "9.2.0"))
+load(pathJoin("hpc-mpich", os.getenv("hpc-mpich_ver") or "3.3.2"))
+
+load("srw_common")
+
+load(pathJoin("nccmp", os.getenv("nccmp_ver") or "1.8.7"))
```

Comment:
The version built with the new stack: 1.8.9 (or 1.9.1)
`load(pathJoin("nccmp", os.getenv("nccmp_ver") or "1.8.9"))`